code/game/atoms.dm

```diff
@@ -185,7 +185,8 @@ directive is properly returned.
 
 /atom/proc/examine(mob/user)
 	var/list/examine_strings = get_examine_text(user)
-	to_chat(user, examine_block(examine_strings.Join("\n")))
+	if(examine_strings)
```

Comment:
there won't ever be a case something calls parent but doesn't run examine strings, so not doing an if check so it runtimes so we can see that there's something wrong is good here, so don't put an if statement here

Reply:
done with log debug because runtimes bad grrrrr